mlbstatsapi/mlb_api.py

```diff
@@ -242,15 +247,17 @@ def get_team_id(self, team_name) -> List[int]:
         >>> mlb.get_team_id("Oakland Athletics")
         [133]
         """
-
-        mlb_data = self._mlb_adapter_v1.get(endpoint='teams')
+        params['sportId'] = sport_id
+        mlb_data = self._mlb_adapter_v1.get(endpoint='teams', ep_params=params)
         team_ids = []
-
+    
         if 'teams' in mlb_data.data and mlb_data.data['teams']:
             for team in mlb_data.data['teams']:
-                if team['name'].lower() == team_name.lower():
-                    team_ids.append(team['id'])
-
+                try:
```

Comment:
Why do you need a try case here? What search key could possibly cause an exception with a if statement simply checking the string?

Reply:
If the key isn't present its going to throw a key error

Reply:
which is why the function was failing in the first place there is a record with no name or fullname from the looks of it

Reply:
@KCNilssen take a look now